### PR TITLE
Cleanup old configuration in readme

### PR DESCRIPTION
**Pull request by Oxerror**
Remove old configuration section in Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,14 +49,3 @@ The following example shows the container used for linting the repository:
...
-
-### Configuration
-
-These environment variables are supported:
-
-| Environment variable   | Default value        | Description                                                      |
-|------------------------|----------------------|------------------------------------------------------------------|
-| MARKDOWNLINT_VERSION   | `0.39.0`             | Version of markdownlint-cli installed in the image.              |
-
-[markdownlint]: https://github.com/igorshubovych/markdownlint-cli
```
Comment (pascalberger) [inline README.md:61]: Don't remove link

File: README.md [hunk 2]
```diff
@@ -49,14 +49,3 @@ The following example shows the container used for linting the repository:
...
-[Azure Pipelines container jobs]: https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases
```
Comment (pascalberger) [inline README.md:62]: Don't remove link

**Review by pascalberger — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Oxerror) [reply README.md:62]: added links back

File: README.md (hunk 1 above)
Comment (Oxerror) [reply README.md:61]: added link back

**Review by pascalberger — APPROVED** [review]
LGTM

**Pull request merged**